packages/snfoundry/scripts-ts/deploy.ts

```diff
@@ -4,21 +4,64 @@ import {
   exportDeployments,
   deployer,
 } from "./deploy-contract";
+import { green } from "./helpers/colorize-log";
 
+/**
+ * Deploy a contract using the specified parameters.
+ *
+ * @example (deploy contract with contructorArgs)
+ * const deployScript = async (): Promise<void> => {
+ *   await deployContract(
+ *     {
+ *       contract: "YourContract",
+ *       contractName: "YourContractExportName",
+ *       constructorArgs: {
+ *         owner: deployer.address,
+ *       },
+ *       options: {
+ *         maxFee: BigInt(1000000000000)
+ *       }
+ *     }
+ *   );
+ * };
+ *
+ * @example (deploy contract without contructorArgs)
+ * const deployScript = async (): Promise<void> => {
+ *   await deployContract(
+ *     {
+ *       contract: "YourContract",
+ *       contractName: "YourContractExportName",
+ *       options: {
+ *         maxFee: BigInt(1000000000000)
+ *       }
+ *     }
+ *   );
+ * };
+ *
+ *
+ * @returns {Promise<void>}
+ */
 const deployScript = async (): Promise<void> => {
-  await deployContract(
-    {
-      owner: deployer.address, // the deployer address is the owner of the contract
+  await deployContract({
```

Comment:
lets keep this to the bare minimum, just `deploy(contractName, constructor )`, we already have examples above.